helm/templates/nginx-deployment.yaml

```diff
@@ -43,9 +43,9 @@ spec:
           - name: MINIO_PROXY_PASS_HOST
             value: "minio-server"
           - name: DISABLE_HTTP
-            value: {{ $.Values.minio.nginx.disableHttp | quote }}
+            value: {{ $.Values.minio.nginx.DISABLE_HTTP | quote }}
           - name: DISABLE_HTTPS
-            value: {{ $.Values.minio.nginx.disableHttps | quote }}
+            value: {{ $.Values.minio.nginx.DISABLE_HTTPS | quote }}
```

Comment:
Does this produce the expected result? what happens when you quote a boolean in Helm?

Reply:
I believe so. The testing will reveal the discrepancies if any.
The values of environment variables must be quoted.

Reply:
yes, but what happens when you quote a boolean value? how does helm parse it?